node-module-system/calculate.js

```diff
@@ -0,0 +1,23 @@
+import add from './add.js';
+
+import subtract from './subtract.js';
+
+import divide from './divide.js';
+
+import multiply from './multiply.js';
+
+if (process.argv[3] === 'plus') {
+  console.log('result:', add(Number(process.argv[2]), Number(process.argv[4])));
+}
+
+if (process.argv[3] === 'minus') {
+  console.log('result:', subtract(Number(process.argv[2]), Number(process.argv[4])));
+}
+
+if (process.argv[3] === 'times') {
+  console.log('result:', multiply(Number(process.argv[2]), Number(process.argv[4])));
+}
+
+if (process.argv[3] === 'over') {
```

Comment:
The best way to organize your code is to do all the handling of the user input upfront in one place, and all the output to the user at the bottom in one place. Then all the logic is in the middle.

This gives you more flexibility to change your mind about the user interface. For example, you might decide to write a GUI instead of a CLI. It also makes the code more organized and understandable.

Reply:
Noted on all of those.  Will make changes.